src/main/java/jenkins/branch/OrganizationFolderViewHolder.java

```diff
@@ -226,7 +227,10 @@ public ViewImpl(ViewGroup owner, @NonNull SCMSourceCategory category) {
          */
         @Override
         public String getDisplayName() {
-            return category.getDisplayName().toString() + " (" + getItems().size() + ")";
+            //Only used to get a count and to prevent poor performance in cases with large number of items
```

Comment:
How poor is the performance?

Reply:
In the most pathological case(1k Branches with Pull Requests), in can take 15 seconds or so.

This is only for users with an authorization strategy